### PR TITLE
Changed how manual events are added to a sketch.

### PR DESCRIPTION
**Pull request by kiddinn**
This PR allows the manually added events to include attributes and tags.

### CODE REVIEW

File: api_client/python/timesketch_api_client/sketch.py [hunk 1]
```diff
@@ -1255,11 +1263,31 @@ def add_event(self, message, timestamp, timestamp_desc):
             raise RuntimeError(
                 'Unable to add an event to an archived sketch.')
 
+        if tags is None:
+            tags = []
+
+        if not isinstance(tags, list):
+            raise ValueError('Tags needs to be a list.')
+
+        if any([not isinstance(x, str) for x in tags]):
```
Comment (berggren) [inline api_client/python/timesketch_api_client/sketch.py:1272]: s/x/tag

File: api_client/python/timesketch_api_client/sketch.py [hunk 2]
```diff
@@ -1240,13 +1240,21 @@ def search_by_label(self, label_name, as_pandas=False):
...
-    def add_event(self, message, timestamp, timestamp_desc):
+    def add_event(
+            self, message, timestamp, timestamp_desc, attributes=None,
+            tags=None):
         """Adds an event to the sketch specific timeline.
 
         Args:
             message: A string that will be used as the message string.
-            timestamp: Micro seconds since 1970-01-01 00:00:00.
+            timestamp: A string with the timestamp of the message.
```
Comment (berggren) [inline api_client/python/timesketch_api_client/sketch.py:1250]: Is this a timestamp, or a datetime string?

File: api_client/python/timesketch_api_client/sketch.py (hunk 1 above)
Comment (kiddinn) [reply api_client/python/timesketch_api_client/sketch.py]: done

File: api_client/python/timesketch_api_client/sketch.py (hunk 2 above)
Comment (kiddinn) [reply api_client/python/timesketch_api_client/sketch.py]: done

**Pull request merged**